### PR TITLE
[Chore : # 162727507] : Setup sass and css

### PR DESCRIPTION
**Pull request by habibaudu**
#### What does this pull request do?
- Setup  Sass and Css

#### Description of task to be completed?
- [x] Install node-sass
- [x] install css-loader
- [x] install  sass-loader
- [x] install style-loader
- [x] include loaders in wepack.common.js
- [x] create style folder and a sample index.css file to test

#### How should this be tested?
- On the terminal, run `npm start:dev` 
- On the page, you should see a horse blood color for the header

#### What are the relevant pivotal tracker stories?
[#162727506](https://www.pivotaltracker.com/story/show/162727506)

### CODE REVIEW

File: src/style/index.css [hunk 1]
```diff
@@ -0,0 +1,4 @@
+div {
+  background-color:#8c001c;
+  padding: 10px;
+}
```
Comment (andela-mabdussalam) [inline src/style/index.css]: Add newline here.

Comment (habibaudu) [reply src/style/index.css]: No

**Comment by andela-mabdussalam** [comment]
Add a .scss file with styles that are used so that we can validate that the styles are pre-processed.

File: src/style/index.css (hunk 1 above)
Comment (habibaudu) [reply src/style/index.css]: its a css file. my eslint didn't pick it

**Comment by habibaudu** [comment]
> Add a .scss file with styles that are used so that we can validate that the styles are pre-processed.

Ok. doing that right away

File: src/style/index.css (hunk 1 above)
Comment (habibaudu) [reply src/style/index.css]: Ok.. sorry, didn't get your comment at first

**Comment by andela-mabdussalam** [comment]
@habibaudu Remove the css file, webpack would compile to css.

**Comment by habibaudu** [comment]
> @habibaudu Remove the css file, webpack would compile to css.

Ok. Doing that right away

**Comment by andela-mabdussalam** [comment]
@habibaudu Let us add style linting for the css files too.

Specifically, add the stylelint config plugin to webpack and a `stylelintrc` file to the root directory.

**Comment by habibaudu** [comment]
> @habibaudu Let us add style linting for the css files too.
> 
> Specifically, add the stylelint config plugin to webpack and a `stylelintrc` file to the root directory.

Ok. will do that right away.

File: .stylelintrc [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
+  "rules": {
```
Comment (andela-mabdussalam) [inline .stylelintrc:7]: Let us add `stylelint-config-standard` and extend it here.

File: webpack/webpack.common.js [hunk 3]
```diff
@@ -32,6 +32,14 @@ plugins:[
     {
       template: './src/index.html'
     }
-  )
+  ),
+  new StyleLintPlugin({
+    configFile: '.stylelintrc',
+    context: 'src',
+    files: '**/*.css',
```
Comment (andela-mabdussalam) [inline webpack/webpack.common.js]: Should be files that end with scss

File: webpack/webpack.common.js [hunk 4]
```diff
@@ -32,6 +32,14 @@ plugins:[
...
+    failOnError: false,
```
Comment (andela-mabdussalam) [inline webpack/webpack.common.js]: This is false by default, let us set it to true.

File: .stylelintrc (hunk 2 above)
Comment (habibaudu) [reply .stylelintrc:7]: ok. doing that right away

File: webpack/webpack.common.js (hunk 3 above)
Comment (habibaudu) [reply webpack/webpack.common.js]: ok doing that right away

File: webpack/webpack.common.js (hunk 4 above)
Comment (habibaudu) [reply webpack/webpack.common.js]: ok. will change to true

**Review by andela-mabdussalam — APPROVED** [review]

**Pull request merged**